source/Ship.cpp

```diff
@@ -1925,6 +1929,7 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 				discharge += scale * attributes.Get("turning discharge");
 				corrosion += scale * attributes.Get("turning corrosion");
 				ionization += scale * attributes.Get("turning ion");
+				scrambling += scale * attributes.Get("turning scrambling");
```

Comment:
Here and with a few other attributes: if you look to burn damage, it's sometimes called burn and sometimes called burning depending on what fits better. Leak damage is similar, sometimes being leak and sometimes being leakage, or ion damage with ion and ionization. We should do the same here. That is, don't just use "scrambling" across the board. Look for what fits better between scrambling and scramble. In this case, I think "turning scramble" works better than "turning scrambling."

Reply:
I cant really tell which fits better but I'll try to look at what happens for leakage and copy it

Reply:
I can comment on each instance if you need.

Reply:
I renamed: 
turning (scramble), thrusting, afterburner, scrambling spark -> scramble

Do tell me if others need a change

Reply:
Could you put all the new attribute names in the PR description?

Reply:
Done